pkg/webui/console/views/device-add/wizard/network-settings-form/index.js

```diff
@@ -68,6 +71,13 @@ const NetworkSettingsForm = props => {
     setResetsFCnt(checked)
   }, [])
 
+  const fCntWidthEncode = React.useCallback(value => {
```

Comment:
You can remove `fCntWidthEncode` and `fCntWidthDecode` from the component. There are no dependencies at all.

Reply:
```js
const fCntWidthEncode = value => value === FRAME_WIDTH_COUNT.SUPPORTS_32_BIT

const fCntWidthDecode = value => value ? FRAME_WIDTH_COUNT.SUPPORTS_32_BIT : FRAME_WIDTH_COUNT.SUPPORTS_16_BIT

const NetworkSettingsForm = props => {
...
```